src/Kovan-DssSpell.sol

```diff
@@ -88,222 +87,209 @@ contract SpellAction {
         address ILK_REGISTRY = CHANGELOG.getAddress("ILK_REGISTRY");
         address FAUCET       = CHANGELOG.getAddress("FAUCET");
 
+        // Set the global debt ceiling
+        // + 20 M to fix an error introduced in previous spell not counting the previous 20M of ETH-B (before introducing IAM)
+        // + 10 M for AAVE-A
+        // +  3 M for UNIV2DAIETH-A
+        VatAbstract(MCD_VAT).file("Line", VatAbstract(MCD_VAT).Line() + 23 * MILLION * RAD + 10 * MILLION * RAD);
```

Comment:
Nit: it's a little inconsistent to express as 20 + 10 + 3 in the comment but 23 + 10 in the code.

Reply:
fixed